app/modules/import-export/assets/js/shared/process-failed-dialog/process-failed-dialog.js

```diff
@@ -60,6 +60,14 @@ const messagesContent = {
 		title: __( 'Couldn’t install the Kit', 'elementor' ),
 		text: __( 'The Kit includes plugins you don’t have permission to install. Contact your site admin to change your permissions.', 'elementor' ),
 	},
+	'domdocument-missing': {
+		title: __( 'Couldn’t install the Kit', 'elementor' ),
+		text: __( 'You must have DOMDocument installed', 'elementor' ),
+	},
```

Comment:
You have a mixup of two PRs. Please take these changes out of this one.
```suggestion
```

Reply:
This branch s coming out from the other branch...
Should i rebase him?